src/main/java/de/telran/service/CsvFileService.java

```diff
@@ -20,13 +19,15 @@
         }
     }
 
+
     public void saveImageAsFile(BufferedImage image) {
 
         try {
-            ImageIO.write(image, "jpg",new File("/Users/slukichev/Downloads/images/img_"+image.hashCode()+".jpg"));
+            ImageIO.write(image, "jpg",new File("C:\\Users\\Tatjana Kufeld\\Desktop\\Project\\CommandLineImageProcessor\\test_images.csv\"+image.hashCode()"));
```

Comment:
the path to a folder where you save images should be parametrized and stored in the application.properties file.